src/server/game/Server/Packets/ChatPackets.h

```diff
@@ -45,6 +45,17 @@ namespace WorldPackets
 
             uint32 EmoteID = 0;
         };
+
+        class ChatServerMessage final : public ServerPacket
+        {
+        public:
+            ChatServerMessage() : ServerPacket(SMSG_CHAT_SERVER_MESSAGE, 4 + 2) { }
```

Comment:
is 2 the most common size of StringParam ?

Reply:
This value was taken from original commit - https://github.com/TrinityCore/TrinityCore/commit/fc7014628af364911de75eebcac2034676292b30

Reply:
```suggestion
            ChatServerMessage() : ServerPacket(SMSG_CHAT_SERVER_MESSAGE, 4 + 20) { }
```
StringParam is usually the shutdown countdown string with minutes+seconds in it (who does hour long shutdowns?)